crates/experimentation_client/src/interface.rs

```diff
@@ -208,9 +221,62 @@ pub extern "C" fn expt_get_satisfied_experiments(
         Err(err) => return error_block(err),
     };
 
+    let prefix_list = if filter_prefix.is_null() {
+        None
+    } else {
+        let prefix_list = match cstring_to_rstring(filter_prefix) {
+            Ok(filter_string) => filter_string.split(',').map(String::from).collect(),
+            Err(err) => {
+                update_last_error(err);
+                return std::ptr::null_mut();
+            }
+        };
+        Some(prefix_list)
+    };
+
+    let local = task::LocalSet::new();
+    let experiments = local.block_on(&Runtime::new().unwrap(), unsafe {
+        (*client).get_satisfied_experiments(&context, prefix_list)
+    });
+    let experiments = match serde_json::to_value(experiments) {
```

Comment:
`let experiments = serde_json::to_value(experiments).map_err(|err| error_block(err.to_string()))?;`

Reply:
Can't do this since it is not result type.